tests/functional/generate_pages.py

```diff
@@ -43,21 +43,21 @@
     'Defaults to ".."',
 )
 
-SPEC_PAGE_HEADER = "const helpers = require('../helpers');\n\n"
+SPEC_PAGE_HEADER = "import helpers from '../helpers';\n\n"
 
 SPEC_PAGE_IMPORT = Template(
-    r"""const ${pageName}Page = require('../generated_pages/${pageDir}/${pageFile}');
+    r"""import ${pageName}Page from '../generated_pages/${pageDir}/${pageFile}';
 """
 )
 
 SPEC_EXAMPLE_TEST = Template(
     r"""
-describe('Example Test', function() {
-  beforeEach('Load the survey', function() {
-    browser.openQuestionnaire('${schema}');
+describe("Example Test", () => {
+  beforeEach"Load the survey", () => {
```

Comment:
Is this how beforeEach is called in ES6?

Reply:
Fixed the bracket.